spec/helpers/categories_helper_spec.rb

```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 describe CategoriesHelper do
-  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I would just delete the file rather than commenting out the pending notification. The reason we don't use generators in the course is because it litters your app with unused files like these.